RELEASE-NOTES.md

```diff
@@ -1,5 +1,9 @@
 # Wikibase DataModel release notes
 
+## Version 9.0.1 (2018-11-09)
```

Comment:
Shouldn't this rather be `9.1.0`? I don't think this is technically fixing a bug, given the (reverted) change was intended?

Reply:
The revert was meant to undo to 8.0 changes and go back to the 7.5 behavior. I fucked up. So yeah, this is a bugfix. If it is not a bugfix, then it is a breaking change (10.0), which we have incentive to avoid (costly).

Reply:
Ok, makes sense